### DISCUSSION TITLE
Trouble saving the file in X directory

### DISCUSSION THREAD

**Discussion by ezee969** [py-pdf/fpdf2#289] [Q&A] [answered]
Hello, can i get some help? im triying to save the PDF file in my desktop using : output ( "pdf_name" , C:\Users\Myname ).
And im getting the following error: 

SyntaxError: (unicode error) 'unicodeescape' codec can't decode bytes in position 2-3: truncated \uXXXX escape

Anyone knows how to fix it?

EDIT; i tried using  output ( "pdf_name" , r "C:\Users\Myname" ) and  output ( "pdf_name" , C:/Users/Myname ) and i dont get the error anymore but it saves the file in the current folder im working with python,

**Comment by Lucas-C** [comment, marked as answer]
Under Windows, you should be able to save your file this way:

```
pdf.output(r"C:\Users\MyName\pdf_name.pdf")
```

If you don't know the `r` string prefix in Python, you may want to check this explanation:
https://stackoverflow.com/questions/4780088/what-does-preceding-a-string-literal-with-r-mean

The error you witness could be caused by other parts of your code...
Could you provided some [minimal code reproducing your issue](https://stackoverflow.com/help/minimal-reproducible-example) please, so that I can help you further?

**Reply by ezee969** [reply]
thanks it worked that way
